src/main/java/co/cask/PluginJSON.java

```diff
@@ -116,7 +116,7 @@ public void execute()  throws MojoExecutionException {
 
     // Checks if the widget path specified by the user is a directory or not. If the path specified is not
     // not directory, then we would bail from further processing.
-    if (! widgetDirectory.isDirectory()) {
+    if (!widgetDirectory.isDirectory()) {
```

Comment:
you can remove this from here since it is checking directories at initialization